docs/labs.md

```diff
@@ -67,3 +67,9 @@ An implementation of [MSC2241](https://github.com/matrix-org/matrix-doc/pull/224
 ## New user info panel (`feature_user_info_panel`)
 
 A new implementation of the user & member info panel, designed to share more code between showing community members & room members.
+
+## Cross-signing (`feature_cross_signing`)
+
+Cross-signing ([MSC1756](https://github.com/matrix-org/matrix-doc/pull/1756))
+improves the device verification experience by allowing you to verify a user
```

Comment:
minor nit: we should probably humanize this doc a bit.

Reply:
Yeah... I haven't been sure what the doc's goal is entirely I guess... Do we want to offer a very quick setting name to one sentence summary for very slightly more understanding, or is it meant to properly explain the feature?

If it's meant to properly explain the feature, then maybe it should live somewhere else so that we don't just delete the documentation when the flag goes away.

Many similar sounding questions ensue...

Reply:
I believe it was originally added as a way for riot admins to know what the flags are so they can pick and choose which ones they want. In practice I think it's become a bunch of helpful headers (feature name + flag) with some not-as-helpful longer form text. Maybe we just convert it to a list?

I do see some value in having a description here, but I'm not sure what the description should be. It definitely shouldn't be an exhaustive description of the feature because that's best served elsewhere (such as in user guides we don't have). It could be a high level overview of what to expect by turning it on?

Reply:
Hmm yeah, some small description does seem helpful to have outside of the app UI for configuration, archaeology, etc. but also unsure of the right balance.

Added a retro item for this; I'll merge this text for now.